core/tests/protractor/publicationAndGallery.js

```diff
@@ -37,23 +37,27 @@ describe('Gallery view', function() {
     var LANGUAGE_FRANCAIS = 'français';
     var LANGUAGE_DEUTSCH = 'Deutsch';
 
-    users.createModerator('varda@example.com', 'Varda');
-    users.createUser('feanor@exmple.com', 'Feanor');
-    users.createUser('celebrimbor@example.com', 'Celebrimbor');
-    users.createUser('earendil@example.com', 'Earendil');
-
-    users.login('feanor@exmple.com');
```

Comment:
Please keep the existing names (with the new suffixes) in this file as they are thematic.